support/Machines/Mbot.yaml

```diff
@@ -0,0 +1,13 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+Machine:
+  Name: Mbot
+  Description: |
+    Supercomputer at Cornell hosted by Nils Deppe.
+    More information:
+    https://www.cac.cornell.edu/wiki/index.php?title=MBOT_Cluster
```

Comment:
I'm trying to collect detailed info here: https://github.com/sxs-collaboration/WelcomeToSXS/wiki/Mbot so maybe better to point to that?